chart/templates/node/daemonset.yaml

```diff
@@ -74,7 +74,7 @@ spec:
         {{- end }}
       containers:
         - name: csi-node-driver-registrar
-          image: {{ tpl .Values.node.image.csiNodeDriverRegistrar.name . }}
+          image: {{ tpl .Values.node.image.csiNodeDriverRegistrar.name . }}:{{ .Values.node.image.csiNodeDriverRegistrar.tag }}
```

Comment:
Sorry I didn't express myself correctly. I think this is not backward compatible, we should check if the tag is set, and only then append `:{{ tag }}`. Right now we might end up with `{{ name }}:v2.9.0:`.

Reply:
I am going to push the required fix.